lib/oeqa/selftest/cases/updater.py

```diff
@@ -655,7 +655,14 @@ def qemu_launch(efi=False, machine=None, imagename=None):
     args.dir = 'tmp/deploy/images'
     args.efi = efi
     args.machine = machine
-    args.kvm = None  # Autodetect
+    get_bb_var('DEPLOY_DIR_IMAGE')
```

Comment:
What is the benefit of getting `DEPLOY_DIR_IMAGE` but not using it?

Reply:
Oh I think that is a copy paste issue. Will update the merge request.